post_assets.py

```diff
@@ -0,0 +1,75 @@
+import sys
+import os
+import csv
+import django
+
+project_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(project_dir)
+django.setup()
+from core.models.asset import Asset, AssetModelNumber   # noqa
+
+
+def bulk_create_asset():
+    with open('assets.csv', 'r', ) as f:
```

Comment:
can we change this to prompt the user to enter the file name? (without the file extension)..

Reply:
Okay.  I will look into that.